_maps/map_files/CTF/downtown.dmm

```diff
@@ -2126,9 +2126,6 @@
 /obj/structure/rack{
 	resistance_flags = 64
 	},
-/obj/item/clothing/head/beret/sec/navywarden{
```

Comment:
Could you replace this with a recolored blue beret rather than just leaving this empty?